src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -139,7 +190,10 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
         api.getFileAPI().setFileDescription(
             file,
             newDescription,
-            this.descriptionChangeSuccessCallback,
+            () => {
+                // Refetch the file from cache
+                this.fetchFile();
```

Comment:
Do we need to fetch the file object again after updating the description? Or can we just set state with the response as we were before?

Reply:
This is true, the response actually is the cache value. Putting it back